### ISSUE TITLE
No way to test animated content

### ISSUE THREAD

**Issue by gsnedders** [w3c/web-platform-tests#7759]
Realistically, we're never going to do frame-by-frame testing of animation. From memory, most browsers have APIs to pause animations at a certain time offset and then screenshot that.

**Comment by birtles** [comment]
Yeah, we do that either using the Web Animations API, or using various combinations of step timing functions and negative delays to seek the animation to different points. More commonly, though, we just do script-based tests and call `getComputedStyle`.

**Comment by JKereliuk** [comment]
@birtles Do you know of any existing manual tests that require this?

**Comment by birtles** [comment]
I don't know for sure but there may be some in css-animations/css-transitions.

Ultimately, though, I don't think there is any need for manual tests of animations. If there are manual tests in css-animations/css-transitions we should double-check the same functionality is covered by automated tests and then remove them. I am very gradually rewriting all the tests in css-animations / css-transitions so I'll do that when I get to them.

**Comment by gsnedders** [comment]
Certainly there are SVG manual tests which test animation, somewhere in `svg/`.
